src/rules/indentation/README.md

```diff
@@ -120,7 +120,6 @@ For example, with `2`:
 The following patterns are considered warnings:
 
 ```css
-```css
```

Comment:
Sorry, I forgot to mention this. I just noticed that this was a mistake and corrected it. The extra ```css was getting printed in the rendered output:

![screen shot 2015-10-01 at 8 42 27 am](https://cloud.githubusercontent.com/assets/6687627/10225631/97482f7a-6818-11e5-89bd-565ef0b0c6c0.png)

Thanks!